catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClient.java

```diff
@@ -339,23 +354,23 @@ protected String getSortProperty(QueryRequest request, SolrQuery query) {
         return resolver.getSortKey(sortProperty);
     }
 
-    private ResultImpl createResult(SolrDocument doc, String sortProperty)
+    private ResultImpl createResult(SolrDocument doc)
             throws MetacardCreationException {
         ResultImpl result = new ResultImpl(createMetacard(doc));
 
         if (doc.get(RELEVANCE_SORT_FIELD) != null) {
-            if (Result.RELEVANCE.equals(sortProperty)) {
-                result.setRelevanceScore(((Float) (doc.get(RELEVANCE_SORT_FIELD))).doubleValue());
-            } else if (Result.DISTANCE.equals(sortProperty)) {
-                Object distance = doc.getFieldValue(RELEVANCE_SORT_FIELD);
+            result.setRelevanceScore(((Float) (doc.get(RELEVANCE_SORT_FIELD))).doubleValue());
+        }
 
-                if (distance != null) {
-                    LOGGER.debug("Distance returned from Solr [{}]", distance);
-                    double convertedDistance = degreesToMeters(Double.valueOf(distance.toString()));
+        if (doc.get(DISTANCE_SORT_FIELD) != null) {
+            Object distance = doc.getFieldValue(DISTANCE_SORT_FIELD);
 
-                    LOGGER.debug("Converted distance into meters [{}]", convertedDistance);
-                    result.setDistanceInMeters(convertedDistance);
-                }
+            if (distance != null) {
+                LOGGER.debug("Distance returned from Solr [{}]", distance);
+                double convertedDistance = Double.valueOf(distance.toString()) * 1000;
+
+                LOGGER.debug("Converted distance into meters [{}]", convertedDistance);
```

Comment:
It made more sense when one value was in radians and the other was in meters.  I removed the converted distance log message.